x-pack/plugins/secops/public/components/timeline/body/renderers/google_link.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexItem, EuiLink } from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+const LinkFlexItem = styled(EuiFlexItem)`
+  margin-left: 6px;
+`;
+
+export const GoogleLink = pure(({ link, value }: { link: string; value: string }) => (
+  <LinkFlexItem grow={false}>
```

Comment:
optional / discussion: In a way, the `LinkFlexItem` is encoding `margin-left: 6px;` as part of the "implicit API" of this component, in the sense that any consumer of `<GoogleLink />` will have this margin applied to it, which may not always be desired.

If that sounds reasonable, perhaps the same thing could be said of wrapping the `EuiLink` in a  `EuiFlexItem`?  The API doesn't suggest this component must live inside a flex container, but in a way, that's what the implementation might be suggesting.

Reply:
Yea, I am removing that right now so it's not wrapped in that anymore so it's re-usable.